buildtools_test.go

```diff
@@ -1,40 +1,17 @@
 package main
 
 import (
-	"encoding/json"
 	"fmt"
-	"github.com/jfrog/gocmd/executers"
-	gofrogcmd "github.com/jfrog/gofrog/io"
-	"github.com/jfrog/jfrog-cli-go/artifactory/commands/generic"
-	"github.com/jfrog/jfrog-cli-go/artifactory/commands/gradle"
-	"github.com/jfrog/jfrog-cli-go/artifactory/commands/mvn"
-	"github.com/jfrog/jfrog-cli-go/artifactory/commands/pip"
-	"github.com/jfrog/jfrog-cli-go/artifactory/spec"
-	"github.com/jfrog/jfrog-cli-go/artifactory/utils"
-	piputils "github.com/jfrog/jfrog-cli-go/artifactory/utils/pip"
-	"github.com/jfrog/jfrog-cli-go/inttestutils"
+	"os"
+	"path/filepath"
+	"testing"
+
 	"github.com/jfrog/jfrog-cli-go/utils/cliutils"
 	"github.com/jfrog/jfrog-cli-go/utils/config"
-	"github.com/jfrog/jfrog-cli-go/utils/ioutils"
 	"github.com/jfrog/jfrog-cli-go/utils/tests"
 	"github.com/jfrog/jfrog-client-go/artifactory/buildinfo"
-	"github.com/jfrog/jfrog-client-go/artifactory/services/go"
-	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
-	"github.com/jfrog/jfrog-client-go/utils/log"
-	"github.com/jfrog/jfrog-client-go/utils/version"
-	"io"
-	"io/ioutil"
-	"os"
-	"os/exec"
-	"path"
-	"path/filepath"
-	"strconv"
-	"strings"
-	"testing"
 )
 
-const DockerTestImage string = "jfrog_cli_test_image"
-
 func InitBuildToolsTests() {
```

Comment:
I'm not sure the *buildtools_test.go* file is needed anymore, because all build tools have their own test files now.

Reply:
You're right, I will merge this file with main_test.go.